backend/infrahub/computed_attribute/models.py

```diff
@@ -309,3 +310,84 @@ def from_object(
         )
 
         return definition
+
+
+class ComputedAttrJinja2GraphQLResponse(BaseModel):
+    node_id: str
+    computed_attribute_value: str | None
+    variables: dict[str, Any] = Field(default_factory=dict)
+
+
+class ComputedAttrJinja2GraphQL(BaseModel):
+    node_schema: NodeSchema = Field(..., description="The node kind where the computed attribute is defined")
+    attribute_schema: AttributeSchema = Field(..., description="The computed attribute")
+    variables: list[str] = Field(..., description="The list of variable names used within the computed attribute")
+
+    def render_graphql_query(self, query_filter: str, filter_id: str) -> str:
+        return f"""query {{
```

Comment:
I think it would be easier and safer to use a GraphQL query object here instead of using a string 
https://github.com/opsmill/infrahub-sdk-python/blob/stable/infrahub_sdk/graphql.py#L153

Reply:
I've replace the code to instead use the Query object.